.goreleaser.yml

```diff
@@ -37,12 +37,19 @@ builds:
       - amd64
       - arm
 dockers:
+  # always push a docker image on release for the .Tag version
   - dockerfile: Dockerfile.goreleaser
     binaries:
       - manager
     image_templates:
-      - "kudobuilder/controller:latest"
       - "kudobuilder/controller:{{ .Tag }}"
+  # only update the docker :latest for a full release (not RC)
+  - dockerfile: Dockerfile.goreleaser
+    binaries:
+      - manager
+    image_templates:
+      - "kudobuilder/controller:latest"
```

Comment:
Is there anything that uses `:latest`? (AFAICT even if someone manually specifies `--version=latest` it won't work because we prefix the value with letter `v`: https://github.com/kudobuilder/kudo/blob/5ca644e5794851438f2a860caf78331d66d911a2/pkg/kudoctl/kudoinit/options.go#L42

Why not just get rid of the `:latest` altogether?